stormevents/nhc/track.py

```diff
@@ -146,11 +146,15 @@ def from_file(
         path: PathLike,
         start_date: datetime = None,
         end_date: datetime = None,
+        file_deck: ATCF_FileDeck = None,
+        advisories: [ATCF_Advisory] = None,
```

Comment:
`[ATCF_Advisory]` is not a valid type declaration. 

If `advisories` is meant to be a list of `ATCF_Advisory` objects, then:
1. the docstring should be updated accordingly
2. the type declaration should be updated to:
``` python
from typing import List

        advisories: List[ATCF_Advisory = None,
```
In Python 3.9+ you can even omit the import and declare it as:
``` python
        advisories: list[ATCF_Advisory] = None,
```

That being said, if an explicit `list` is not required by the code, then `typing.Iterable` is probably a better choice.

If on the other hand, the docstring in line 157 is accurate then the declaration should be:
``` python
        advisories: ATCF_Advisory = None,
```

Reply:
@SorooshMani-NOAA @zacharyburnett Which you think it is?

Reply:
@WPringle I would also update the docstring. Maybe something like:

``` diff
- :param advisories: ATCF advisory type; one of ``BEST``, ``OFCL``, ``OFCP``, ``HMON``, ``CARQ``, ``HWRF``
+ :param advisories: list of ATCF advisory types; valid choices are: ``BEST``, ``OFCL``, ``OFCP``, ``HMON``, ``CARQ``, `HWRF``
 ```